src/main/java/com/example/demo/config/SecurityConfig.java

```diff
@@ -37,7 +37,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http)
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(
                         auth -> auth
-                                .requestMatchers("/auth/**",
+                                .requestMatchers("/users/**",
```

Comment:
why do you change it?

Reply:
@RestController
@RequestMapping("/users") should i change naming of this value 
@Validated
@RequiredArgsConstructor
public class AuthenticationController {